infobind/bind.go

```diff
@@ -0,0 +1,34 @@
+package infobind
+
+import (
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/sessionctx"
+)
+
+var _ Manager = (*BindManager)(nil)
+
+// BindManager use to manage both global bind info and session bind info
+type BindManager struct {
+	is            infoschema.InfoSchema
+	currentDB     string
+	SessionHandle *Handle //session handle
+	*Handle               //global handle
+	copy          bool
+}
+
+type keyType int
+
+func (k keyType) String() string {
+	return "bind-key"
+}
+
+// Manager is the interface for providing bind related operations.
+type Manager interface {
```

Comment:
Why it is an empty interface?

Reply:
the method will be added in next pull request

Reply:
ok, I'll delete it